### PR TITLE
Aleth Python wrapper

### PR DESCRIPTION
**Pull request by chfast**
Add Python script wrapping aleth with JSON-RPC HTTP server

### CODE REVIEW

**Comment by gumb0** [comment]
Does it print any reasonable error message when run without args and aleth is not found?

I'd output the string with expected parameters in this case. Also maybe add `--help`

File: scripts/aleth.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+DEFAULT_ALETH_EXEC = '/usr/bin/aleth'
+
+parser = argparse.ArgumentParser(add_help=False)
+parser.add_argument('--aleth-exec', default=DEFAULT_ALETH_EXEC)
+parser.add_argument('--rpc', action='store_true')
+wrapper_args, aleth_args = parser.parse_known_args()
+
+try:
+    if wrapper_args.rpc:
+        run_daemon()
```
Comment (gumb0) [inline scripts/aleth.py]: Will it still output log messages when run as a daemon

**Comment by chfast** [comment]
> Does it print any reasonable error message when run without args and aleth is not found?
> 
> I'd output the string with expected parameters in this case. Also maybe add --help

I added the error message. It also prints the usage message in case of incorrect path to aleth. The `--help` is not supported because I want it to be passed to aleth, so aleth.py options are hidden. aleth.py is going to be used as docker image entry point.

File: scripts/aleth.py (hunk 1 above)
Comment (chfast) [reply scripts/aleth.py]: Yes.

**Pull request merged**